knotx-core/knotx-common/src/main/java/com/cognifide/knotx/dataobjects/ServiceRequest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Knot.x - Reactive microservice assembler - API
+ * Copyright (C) 2016 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.knotx.dataobjects;
+
+import com.google.common.base.Objects;
+
+import io.vertx.core.json.JsonObject;
+
+public class ServiceRequest {
```

Comment:
OK, thanks @marcinczeczko 
Small pull requests are great!
However I am not so advanced to know what are your goal guys.
Will wait for complete version.